nifi-ngsi-bundle/nifi-ngsi-processors/src/main/java/org/apache/nifi/processors/ngsi/ngsi/utils/NGSIUtils.java

```diff
@@ -107,11 +107,11 @@ public ArrayList<Entity> parseNgsiLdEntities(JSONArray content) {
                         for (int j = 0; j < values.length(); j++) {
                             JSONObject value = values.getJSONObject(j);
                             AttributesLD attributesLD = parseNgsiLdAttribute(key, value);
-                            attributes.add(attributesLD);
+                            if (attributesLD.getAttrValue() !=null && attributesLD.getAttrValue().toString() != "null") attributes.add(attributesLD);
```

Comment:
instead of copy / pasting this line in different place, it could be handled in the two `parseNgsiLdXXXAttribute` methods

Reply:
Ok I do this